nativeunwind/elfunwindinfo/elfgopclntab.go

```diff
@@ -544,8 +544,8 @@ func (g *Gopclntab) Symbolize(pc uintptr) (sourceFile string, line uint, funcNam
 	index := sort.Search(g.numFuncs, func(i int) bool {
 		funcPc, _ := g.getFuncMapEntry(i)
 		return funcPc > pc
-	}) - 1
```

Comment:
Is this subtraction needed? Coredump tests are failing because the symbolization does not map anymore, I am wondering if this was added because assuming "if search not found" returns 0